tests/ui/components/identity.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { IdentityComponent } from 'src/ui/home/components/identity'
+import { shallow } from 'enzyme'
+
+describe('Identity component', ()=> {
+  const COMMON_PROPS = {
+    scanning: false,
+    userName: '',
+    phoneNumber: '',
+    emailAddress: '',
+    onUserNameChange: () => null,
+    onPhoneNumberChange: () => null,
+    onEmailAddressChange: () => null,
+    onScannerStart: () => null,
+    onScannerSuccess: () => null,
+    onScannerCancel: () => null
+  }
+  it('matches the snapshot on render', () => {
+
+    const rendered = shallow(<IdentityComponent {...COMMON_PROPS}/>)
+    expect(rendered).toMatchSnapshot()
+  })
+
+  it('matches the snapshot on render with scanning set to true', () => {
+    const props = Object.assign({}, COMMON_PROPS {
```

Comment:
are these props being used in the shallow render below?

Reply:
I see, you are right. I removed the test completely as it is redundant.